pelican/tools/pelican_import.py

```diff
@@ -468,9 +468,8 @@ def get_tumblr_posts(api_key, blogname, offset=0):
                     fmtstr = '<img alt="%s" src="%s" />'
                 content = ''
                 for photo in post.get('photos'):
-                    content += '\n'.join(
-                        fmtstr % (photo.get('caption'),
-                                  photo.get('original_size').get('url')))
+                    content += fmtstr % (photo.get('caption'),
+                                  photo.get('original_size').get('url'))
```

Comment:
I think `'\n'.join(...)` was to keep one image per line if there are multiple and I'd say it's better to keep it in place for nicer output. But the implementation was wrong. So, the whole thing, including `for`, probably should be something like:

```python
content += '\n'.join(fmtstr % (photo.get('caption'),
                               photo.get('original_size').get('url'))
                     for photo in post.get('photos'))
```

PS: Also, you need to lineup your indentation properly to make linter happy.

Reply:
Ahh, that makes sense - I only tested with one image.  Sorry for the slight delay in getting back to this!

I had/have Pylint setup on my end - is there a different linter that is used for Pelican?  If so, can you point me in the direction to get it working properly?  I think I resolved everything in your comment, and have it indented properly, but let me know if there is anything else glaring.

Reply:
At the moment, Pelican uses Flake8. You can see the two linter failures in the CI output: https://github.com/getpelican/pelican/pull/3187/checks#step:5:20

The dev environment setup docs are here: https://docs.getpelican.com/en/latest/contribute.html#setting-up-the-development-environment